app/src/main/java/com/aptoide/uploader/apps/StoreManager.java

```diff
@@ -37,7 +37,10 @@ public Completable upload(List<InstalledApp> apps) {
     return storeNameProvider.getStoreName()
         .flatMapCompletable(storeName -> languageManager.getCurrentLanguageCode()
             .flatMapCompletable(languageCode -> Observable.fromIterable(apps)
-                .flatMapCompletable(app -> uploadManager.upload(storeName, languageCode, app))));
+                .flatMapCompletable(
+                    app -> installedAppsProvider.getInstalledApp(app.getPackageName())
+                        .flatMapCompletable(
+                            installedApp -> uploadManager.upload(storeName, languageCode, app)))));
```

Comment:
here you should pass the installed app instead of the app. The app object does not have the obbs since it is the initial object. Sorry only noticing it now, but now that i tested with the breakpoint i noticed that the obb was still null